commands/fitsio.py

```diff
@@ -8,8 +8,13 @@
 @click.option('-d', '--directory', type=str, help="Specify path to directory of fitsfiles.", required=False)
 @sdi.generator
 def read(directory):
+    # add try (evaluate if the try is efficient)
     if directory == None:
-        paths = glob.glob("{}/*.fits*".format(tkinter.filedialog.askdirectory()))
+        try:
+            paths = glob.glob("{}/*.fits*".format(tkinter.filedialog.askdirectory()))
+        except:
```

Comment:
This looks good Edgar! However, `except` without specifying a kind of error to except is a bad practice. For example, if the directory given to glob does not exist, glob might throw a FileNotFoundError, which would be caught by `except`, resulting in the very strange error message of having a visual file dialog not exist.

A good fix which does catch all tkinter errors might be to separate out the `tkinter.filedialog.askdirectory()` into its own try-catch, leaving the glob part separate.